app-doc/abs-guide/abs-guide-10.ebuild

```diff
@@ -6,8 +6,8 @@ EAPI=7
 DESCRIPTION="An in-depth exploration of the art of shell scripting"
 HOMEPAGE="https://www.tldp.org/LDP/abs/html"
 
-SRC_URI="http://bash.deta.in/abs-guide-final.tar.bz2
-	pdf? ( http://bash.deta.in/abs-guide.pdf )"
+SRC_URI="https://tldp.org/LDP/abs/abs-guide.html.tar.gz
```

Comment:
Have you confirmed this is the same content-wise, by grabbing the version from our mirrors?